package.json

```diff
@@ -10,6 +10,7 @@
   },
   "scripts": {
     "docs": "yarn --cwd packages/docs start",
+    "icons": "yarn --cwd packages/odyssey-icons svgo",
     "lerna-version": "lerna version --no-git-tag-version --force-publish",
     "prelerna-publish": "lerna run build",
```

Comment:
Notice here how we run `lerna run build` before attempting to publish new package versions. This is calling `yarn build` in every package. To ensure all packages build when we're attempting to publish - consider the recommendation above to use the `build` command.

Reply:
See my comment above - this is intended to expose the utility command at a higher level for ease of access to our designers, not be run at build time.